dotcom-rendering/playwright/tests/parallel-3/article.interactivity.e2e.spec.ts

```diff
@@ -0,0 +1,361 @@
+import { devices, expect, test } from '@playwright/test';
+import {
+	expectToBeVisible,
+	expectToExist,
+	expectToNotBeVisible,
+	expectToNotExist,
+} from 'playwright/lib/locators';
+import { disableCMP } from '../../lib/cmp';
+import { waitForIsland } from '../../lib/islands';
+import { loadPage } from '../../lib/load-page';
+import { mockApis } from '../../lib/mocks'; // TODO e2e is this required?
+
+const articleUrl =
+	'https://www.theguardian.com/politics/2019/oct/29/tories-restore-party-whip-to-10-mps-who-sought-to-block-no-deal-brexit';
+
+test.describe('Interactivity', () => {
+	test.describe('Verify elements have been hydrated', () => {
+		test('should open the edition dropdown menu when clicked and hide when expected', async ({
+			context,
+			page,
+		}) => {
+			await disableCMP(context);
+			await loadPage(page, `/Article/${articleUrl}`);
+
+			await waitForIsland(page, 'HeaderTopBar', { status: 'hydrated' });
+			// Open it
+			await page.locator('[data-cy=dropdown-button]').click();
+			await expectToBeVisible(page, '[data-cy=dropdown-options]');
+			// Pressing esc hides it
+			await page.locator('body').press('Escape');
+			await expectToNotBeVisible(page, '[data-cy=dropdown-options]');
+			// Open it again
+			await page.locator('[data-cy=dropdown-button]').click();
+			await expectToBeVisible(page, '[data-cy=dropdown-options]');
+			// Clicking elsewhere in the document hides it
+			await page.locator('h1').first().click();
+			await expectToNotBeVisible(page, '[data-cy=dropdown-options]');
+		});
```

Comment:
Compare to the Cypress version:

https://github.com/guardian/dotcom-rendering/blob/394ba2a3802ca8374bf4f2cafe7ee7e411c40b78/dotcom-rendering/cypress/e2e/parallel-3/article.interactivity.cy.js#L18-L33